apps/root/src/pages/earn/components/strategies-table/components/delayed-withdraw-container/index.tsx

```diff
@@ -1,13 +1,16 @@
-import useDelayedWithdrawalPositions from '@hooks/earn/useDelayedWithdrawalPositions';
 import React from 'react';
 import { ContainerBox } from 'ui-library';
 import PendingDelayedWithdrawals from '../pending-delayed-withdrawals';
 import ReadyDelayedWithdrawals from '../ready-delayed-withdrawals';
+import useEarnPositions from '@hooks/earn/useEarnPositions';
+import { getDelayedWithdrawals } from '@common/utils/earn/parsing';
 
 const DelayedWithdrawContainer = () => {
-  const delayedWithdrawalPositions = useDelayedWithdrawalPositions();
```

Comment:
why the move from a hook to a util?

Reply:
The hook wasn't doing much, and I needed the filtering logic with a provided list. We could keep the hook anyway, but it would be just a 'useSelector'-ish + useMemo, do you think it's worth it in this case?